ocaml-lsp-server/test/e2e-new/documentation.ml

```diff
@@ -0,0 +1,100 @@
+open Test.Import
+
+module Util = struct
+  let call_documentation ~position ?(identifier = None) ?(contentFormat = None)
+      client =
+    let uri = DocumentUri.of_path "test.ml" in
+    let text_document =
+      TextDocumentIdentifier.yojson_of_t @@ TextDocumentIdentifier.create ~uri
+    in
+    let position = Position.yojson_of_t position in
+    let params =
+      `Assoc
+        (("position", position)
```

Comment:
Instead of writing the encoding for the second time, you could just export what you already have and re-use that.

Reply:
It still not possible because custom request are still not exposed. But it is in my current PR.

Reply:
I don't really see the need to wait though. Making the custom request available to the test is less code than copy pasting all this encoding stuff